### PR TITLE
Refactor/New TeskaLabs Logo Favicon

### PR DESCRIPTION
**Pull request by petrKavulok**
### in this PR 
- favicon update with new TL logo

Screenshot 2023-06-28 at 16 45 32
Screenshot 2023-06-28 at 16 46 19

(please don't mind custom title)

### CODE REVIEW

File: public/media/favicon/browserconfig.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	<msapplication>
 		<tile>
 			<square70x70logo src="media/favicon/ms-icon-70x70.png"/>
+			<square144x144logo src="media/favicon/ms-icon-144x144.png"/>
```
Comment (Pe5h4) [inline public/media/favicon/browserconfig.xml:6]: @petrKavulok Why did you add a browser config in demo app with absolute path with the slash at the beginning of the path https://github.com/TeskaLabs/asab-webui/blob/798db86905254f01f9ddcf88af8030f0689e474b/demo/public/media/favicon/browserconfig.xml#L5, but here you dont?

**Review by Pe5h4 — APPROVED** [review]
@petrKavulok Please prior merging, take a look to the comment I wrote and consider it within your decision of merging/updating the xml file

File: public/media/favicon/browserconfig.xml (hunk 1 above)
Comment (petrKavulok) [reply public/media/favicon/browserconfig.xml:6]: honestly, I played along with the rest of config. I'm not sure how to test this
